app/src/pages/common/settingsPage/notificationsTab/modals/addEditNotificationCaseModal/addEditNotificationCaseModal.jsx

```diff
@@ -51,15 +51,8 @@ const messages = defineMessages({
 
 @withModal('addEditNotificationCaseModal')
 @reduxForm({
-  form: 'notificationCaseForm',
-  validate: (
-    { ruleName, recipients, informOwner, launchNames, attributes },
-    { data: { notificationCase, cases } },
-  ) => ({
-    ruleName: commonValidators.createRuleNameValidator(
-      cases.map((item) => ({ name: item.ruleName, ...item })),
-      notificationCase && notificationCase.id,
-    )(ruleName),
```

Comment:
Why did you remove the `ruleName` field validation?
It is required in the 5.8 API.

Reply:
That is great point.
I just cherry-pick commit from 5.7.4 and this form have not ruleName field validation.
I will return this one.